tests/garage/tf/algos/test_ddpg.py

```diff
@@ -97,3 +96,45 @@ def test_ddpg_pendulum(self):
             assert last_avg_ret > 10
 
             env.close()
+
+    @pytest.mark.large
+    def test_ddpg_pendulum_with_decayed_weights(self):
+        """Test DDPG with Pendulum environment and decayed weights.
+
+        This environment has a [-3, 3] action_space bound.
+        """
+        with LocalTFRunner(snapshot_config, sess=self.sess) as runner:
+            env = TfEnv(normalize(gym.make('InvertedPendulum-v2')))
+            action_noise = OUStrategy(env.spec, sigma=0.2)
+            policy = ContinuousMLPPolicy(env_spec=env.spec,
+                                         hidden_sizes=[64, 64],
+                                         hidden_nonlinearity=tf.nn.relu,
+                                         output_nonlinearity=tf.nn.tanh)
+            qf = ContinuousMLPQFunction(env_spec=env.spec,
+                                        hidden_sizes=[64, 64],
+                                        hidden_nonlinearity=tf.nn.relu)
+            replay_buffer = SimpleReplayBuffer(env_spec=env.spec,
+                                               size_in_transitions=int(1e6),
+                                               time_horizon=100)
+            algo = DDPG(
+                env_spec=env.spec,
+                policy=policy,
+                policy_lr=1e-4,
+                qf_lr=1e-3,
+                qf=qf,
+                replay_buffer=replay_buffer,
+                target_update_tau=1e-2,
+                n_train_steps=50,
+                discount=0.9,
+                policy_weight_decay=0.99,
+                qf_weight_decay=0.99,
+                min_buffer_size=int(5e3),
+                exploration_strategy=action_noise,
+            )
+            runner.setup(algo, env)
+            last_avg_ret = runner.train(n_epochs=10,
+                                        n_epoch_cycles=20,
+                                        batch_size=100)
+            assert last_avg_ret > 5
```

Comment:
We get average return of 13 without decayed weights, while we only get 6 when using decayed weights. is this expected? @ryanjulian  @ahtsan

Reply:
i'm not sure. what if you make it much less decayed (e.g. 0.999)

Reply:
the decay parameters are actually regularization constants. 0 means no regularization (no decay), and 1 means max regularization. Setting those parameters to 0.01 gives the desired results.

Reply:
ah thanks. can you make that clear in the docstring?